src/backend/tests/it/utils/pocketic.rs

```diff
@@ -91,6 +91,8 @@ fn init_arg() -> Arg {
     Arg::Init(InitArg {
         ecdsa_key_name: format!("master_ecdsa_public_key_{}", SUBNET_ID).to_string(),
         allowed_callers: vec![Principal::from_text(CALLER).unwrap()],
+        ic_root_key_der: None,
+        supported_credentials: None,
```

Comment:
You provide a test for the upgrade in another PR?

Reply:
What do you mean a test for the upgrade?

Are there already upgrade tests?

Reply:
Sure, for tokens when we added the flag `version` and `enabled`. There are also utils to dowload a particular wasm of the backend and then proceed with an upgrade.

See https://github.com/dfinity/oisy-wallet/tree/main/src/backend/tests/it/upgrade